ZukoAzula2016/src/org/team1540/zukoazula/Portcullis.java

```diff
@@ -21,22 +21,28 @@
     private static final FloatIO leftEncoder = leftGrabMotor.modEncoder().getEncoderPosition();
     private static final FloatIO rightEncoder = rightGrabMotor.modEncoder().getEncoderPosition();
 
-    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.3f);
+    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.3f).negated();
```

Comment:
I negated the control because it used to be inverted. Now, the portcullis arms are controlled similar to the Intake Arm(Pushing the joystick up lifts the portcullis arms, and vice-versa.)